public/data/kalijonn.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "Kali Charan Reddy Jonna",
+  "type": "personal",
+  "bio": "I'm a developer who loves all things Typescript/Javascript",
```

It would look better if you capitalize the `s` in both `JavaScript` and `TypeScript` words, IMO. What do you think? :)
```suggestion
  "bio": "I'm a developer who loves all things TypeScript/JavaScript",
```